handlers/couchbase/couchbase.go

```diff
@@ -0,0 +1,129 @@
+package couchbase
+
+import (
+	"errors"
+
+	"github.com/netflix/rend/common"
+	"github.com/netflix/rend/handlers"
+	gocb "gopkg.in/couchbase/gocb.v1"
+)
+
+var (
+	errNotFound = "key not found"
+)
+
+type Handler struct {
+	client clientWrapper
+}
+
+type clientWrapper interface {
```

Comment:
Where is it used ?

Reply:
Above and in test. Basically it's a wrapper in the client so that it's easier to test